app/api/test/v1/test_meetup.py

```diff
@@ -111,7 +121,8 @@ def test_get_one_meetup(self):
 
     def test_get_all_meetup(self):
             '''tests the for gettting all upcoming  meetups'''
-            response = self.post_meetup()
+            response = self.client.post("/api/v1/meetups", data = json.dumps(self.meetup_2),
+                                                    content_type = "application/json")
             self.assertEqual(response.status_code, 201)
```

Comment:
Can you assert more data other than the status code?

Reply:
Thank you by the application returning the correct status code means that the data in it is correct.